### ISSUE TITLE
Add translations for navigation "Sections" to configs

### ISSUE THREAD

**Issue by OlgaLyubin** [bbc/simorgh#4890]
**Is your feature request related to a problem? Please describe.**
There are currently no translations for a "Sections" word for the navigation menu that should be announced by screen readers. Add these translations to the services configs.

**Describe the solution you'd like**
Translations are in the JIRA ticket 234.

**Describe alternatives you've considered**
N/A

**Testing notes**
Testing will be done as part of the accessibility swarm for navigation (https://github.com/bbc/simorgh/issues/4569).

Dev insight: Will Cypress tests be required or are unit tests sufficient? Will there be any potential regression? etc

- [ ] This feature is expected to need manual testing.

**Additional context**
